string user
---
uint64 startTime
int32 result 
